pkg/build/builder/common.go

```diff
@@ -93,7 +93,7 @@ func buildInfo(build *buildapiv1.Build, sourceInfo *git.SourceInfo) []KeyValue {
 // that the built image can be referred to unambiguously even in the face of
 // concurrent builds with the same name in the same namespace.
 func randomBuildTag(namespace, name string) string {
-	repo := fmt.Sprintf("docker.io/%s/%s", namespace, name)
+	repo := fmt.Sprintf("b.openshift.io/%s/%s", namespace, name)
```

Comment:
can we make this more obviously temporary, like `tmp.build.openshift.io/...`?

Reply:
@liggitt gets all the +1s today.

Reply:
We'll hit length limits at some point.  That's probably fine.